lib/subsetFonts.js

```diff
@@ -728,26 +728,29 @@ async function subsetFonts(
   }
 
   if (!subsetPerPage) {
-    const globalFontUsage = {};
-
-    // Gather all texts
-    for (const htmlAssetTextWithProps of htmlAssetTextsWithProps) {
-      for (const fontUsage of htmlAssetTextWithProps.fontUsages) {
-        if (!globalFontUsage[fontUsage.fontUrl]) {
-          globalFontUsage[fontUsage.fontUrl] = [];
+    const fontUsagesConcat = _.flatMap(
+      htmlAssetTextsWithProps,
+      htmlAssetTextWithProps => htmlAssetTextWithProps.fontUsages
+    );
+    const globalFontUsageMap = fontUsagesConcat.reduce(
+      (fontUsageMap, fontUsage) => {
+        const newFontUsage = fontUsage;
+        if (fontUsage.fontUrl in fontUsageMap) {
+          newFontUsage.text = _.uniq(
+            fontUsageMap[fontUsage.fontUrl].text + fontUsage.text
+          )
+            .sort()
+            .join('');
```

Comment:
Hmm, mental note that we should find a better way achieve the "unique across pages" thing, sorting the output of `_.uniq` breaks when there's multiple non-UTF-16 characters:

```js
require('lodash').uniq('🤗').sort().join('')
'🤗'
require('lodash').uniq('🤗🤞').sort().join('')
'🤗�'
require('lodash').uniq('🤗🤞👊').sort().join('')
'�🡊��'
```

Reply:
This doesn't have the same problem. Not very readable though:
```js
[...new Set([...'🤗🤞👊'])].sort().join('')
```

Reply:
Nice! I think we should go with that, then. Not readable is better than broken :)

Reply:
Implemented your fix here: https://github.com/Munter/subfont/pull/87